envoy/upstream/host_description.h

```diff
@@ -201,6 +201,12 @@ class HostDescription {
    * @return timestamp in milliseconds of when host was created.
    */
   virtual MonotonicTime creationTime() const PURE;
+
+  /**
+   * @return timestamp in milliseconds of when host has transitioned from unhealthy to
```

Comment:
(nit, unit is opaque)

```suggestion
   * @return timestamp of when host has transitioned from unhealthy to
```